tests/test_base_config.ini

```diff
@@ -42,28 +45,181 @@ wind_direction:                ./RME/station_data/wind_direction.csv
 precip:                        ./RME/station_data/precip.csv
 vapor_pressure:                ./RME/station_data/vapor_pressure.csv
 metadata:                      ./RME/station_data/metadata.csv
+stations:                      None
```

Comment:
Good call here. I just redump this config file too and forgot to clean it up. I removed it all so that it is very similar to the original and we still pass. Inicheck won't mess with a value unless it is a default value so if you overwrite it in the raw dictionary it would work just fine. But this will force use to make sure defaults don't change without us knowing.